module-check/src/main/clojure/clojure/core/typed/base_env.clj

```diff
@@ -740,6 +740,9 @@ clojure.string/trim [String -> String]
 clojure.string/trimr [String -> String]
 clojure.string/triml [String -> String]
 
+clojure.template/apply-template [ (t/Vec t/Any ) t/Any (t/List t/Any ) -> t/Any ]
+clojure.template/do-template [ (t/Vec t/Any ) t/Any (t/AnyValue * ) -> t/Any ]
```

Comment:
Please following this formatting: 

```clojure
clojure.template/apply-template [(Vec Any) Any (List Any) -> Any]
```

Note that everything in clojure.core.typed is in scope here.

Also please notice the whitespace here. Whitespace is needed to separate symbols from eachother, but  things like `[(` or `Any]` don't need extra spaces.

Also delete the `clojure.template/do-template` annotation. `do-template` is a macro which are fully expanded by the time type checking happens. Do not annotate macros. You can tell something is a macro if it is defined with defmacro or if it says "macro" in its documentation https://clojure.github.io/clojure/clojure.template-api.html